category/categoryRouter.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const Category = require('./Category.js')
+
+const router = express.Router()
+
+// add endpoints here
+router
+  .route('/')
+  .get((req, res) => {
+    Category.find()
+      .then(categories => {
+        res.status(200).json(categories)
+      })
+      .catch(err => {
+        res.status(500).json({ errorMessage: 'Error getting category', err })
+      })
+  })
+  .post((req, res) => {
+    const { body } = req
+    const category = new Category(body.title)
```

Comment:
The value we're passing into `new Category` should be a JSON object. Currently, we're passing in a string. Simple fix would be to change it to:

```js
const category = new Category(req.body)
```